src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -101,6 +103,9 @@ public class LogManagerService extends PluginService {
     public static final String MULTILINE_PATTERN_CONFIG_TOPIC_NAME = "multiLineStartPattern";
     public static final int DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC = 300;
     private final Object spaceManagementLock = new Object();
+    // TODO: this is the flag to marking the code that used for the feature development, in order to maintain PR
+    //  small and we can modify tests in the following PR.
+    public static final AtomicBoolean ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG = new AtomicBoolean(false);
```

Comment:
This boolean value is only for giving a chance for this small PR to perform some tests without changing the current logic, such that all tests can pass and this small PR can be merged. This is only for serving the purpose of avoiding massive PR, and this static boolean constant will be removed in the end.